server/libs/infra/src/entities/user.entity.ts

```diff
@@ -53,4 +54,10 @@ export class UserEntity {
 
   @OneToMany(() => AssetEntity, (asset) => asset.owner)
   assets!: AssetEntity[];
+
+  @OneToMany(() => PartnerEntity, (partner) => partner.sharedBy)
+  sharedWith!: PartnerEntity[];
+
+  @OneToMany(() => PartnerEntity, (partner) => partner.sharedWith)
+  sharedBy!: PartnerEntity[];
```

Comment:
The wording is a bit tricky because users have `shared` their libraries `with` me, but I've also `shared` my library `with` others.

`user0.sharedWith=user 1, user 2` means user 0 has shared their user 1 and user 2, right?

Reply:
There was a mistake in the field names (`shareBy/shareWith` ). Fixed while I was working on the relations for `PartnerEntity`